coretex/cli/modules/node.py

```diff
@@ -393,24 +395,32 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 def initializeNodeConfiguration() -> None:
     config = loadConfig()
 
-    if isNodeConfigured(config) and not isConfigurationValid(config):
-        raise RuntimeError(f"Invalid configuration. Please run \"coretex node config\" command to configure Node.")
-
     if not isNodeConfigured(config):
         errorEcho("Node configuration not found.")
+
         if isRunning():
-            stopNode = clickPrompt(
-                "Node is already running. Do you wish to stop the Node? (Y/n)",
-                type = bool,
-                default = True,
-                show_default = False
-            )
-
-            if not stopNode:
+            if not click.confirm("Node is already running. Do you wish to stop the Node?", default = True):
                 errorEcho("If you wish to reconfigure your node, use \"coretex node stop\" command first.")
                 return
-
+            
             stop()
 
         configureNode(config, verbose = False)
         saveConfig(config)
+
+    if isConfigurationValid(config):
+        return
+
+    errorEcho("Invalid node configuration found.")
+    if not click.confirm("Would you like to update the configuration?", default = True):
+        raise RuntimeError("Invalid configuration. Please use \"coretex node config\" to update a Node configuration.")
+
+    if isRunning():
```

Comment:
This is being checked twice in this function, can we update it to check only once for this?